svgelements/svgelements.py

```diff
@@ -1989,14 +1989,17 @@ def __hash__(self):
         return hash(self.__key())
 
     def __eq__(self, other):
-        if other is None:
+        if other is None or self.x is None or self.y is None:
             return False
```

Comment:
if self.x and other.x are both None this should actually match. It's a weird point but those matching means the points match. I'm not sure why it would happen but it seems correct.